associations/serializers/association.py

```diff
@@ -131,6 +132,22 @@ def get_my_role(self, obj):
         role = self.context["request"].user.get_role(obj)
         return RoleSerializer(role).data if role else {}
 
+    def get_pages(self, obj):
+        # Get the current user.
+        user = None
+        request = self.context.get("request")
+
+        if request and hasattr(request, "user"):
+            user = request.user
```

Comment:
maybe `user = getattr(self.context.get("request", {}), "user", none)`

Reply:
OK, I think one line is a bit hard to follow, but you do have a point. How about:

```python
request = self.context.get("request", None)
user = getattr(request, "user", None)
```